api/src/zero/module/xx.clj

```diff
@@ -88,9 +100,18 @@
      :cram_ref_fasta       fasta
      :cram_ref_fasta_index (str fasta ".fai")}))
 
+(defn apply-overrides
+  "Return an map like DEFAULTS but overridden with any RAW-ARGS that have FLAG-PREFIX."
+  [defaults flag-prefix raw-args]
+  (merge defaults (->> raw-args
+                       (filter #(str/starts-with? % (str flag-prefix ":")))
+                       (map #(str/split (subs % 3) #"="))
+                       (map (fn [[k v]] [(keyword k) v]))
+                       (into {}))))
+
 (defn make-inputs
   "Return inputs for reprocessing IN-GS into OUT-GS in ENVIRONMENT."
-  [environment out-gs in-gs]
+  [baits-and-targets environment out-gs in-gs]
```

Comment:
Added this as the first argument whenever a function needs to accept it because there are two cases where we use `partial` to slot this in (in other words, it being first is consistent and makes programmatic sense in some of the cases). Happy to put it elsewhere if we think that makes more sense. See below for further comment.